src/procedures/proc_bfs.c

```
@@ -97,14 +97,14 @@ static ProcedureResult Proc_BFS_Invoke
 	GraphContext  *gc  =  QueryCtx_GetGraphCtx();
 
 	if(reltype == NULL) {
-		Delta_Matrix_export(&R, Graph_GetAdjacencyMatrix(gc->g, false));
+		Delta_Matrix_export(&R, Graph_GetAdjacencyMatrix(gc->g));
 	} else {
 		Schema *s = GraphContext_GetSchema(gc, reltype, SCHEMA_EDGE);
```

cppcheck

[error] Common realloc mistake

(memleakOnRealloc)